src/ResourceManager/AzureBatch/Commands.Batch/Commands.Batch.csproj

```diff
@@ -269,12 +195,14 @@
     <Compile Include="Tasks\SetBatchTaskCommand.cs" />
     <Compile Include="Tasks\StopBatchTaskCommand.cs" />
     <Compile Include="NewApplicationPackageException.cs" />
+    <Compile Include="Utils\BatchAadTokenProvider.cs" />
     <Compile Include="Utils\Constants.cs" />
     <Compile Include="Utils\Utils.cs" />
     <Compile Include="ComputeNodes\GetBatchComputeNodeCommand.cs" />
     <Compile Include="JobSchedules\GetBatchJobScheduleCommand.cs" />
     <Compile Include="JobSchedules\NewBatchJobScheduleCommand.cs" />
     <Compile Include="JobSchedules\RemoveBatchJobScheduleCommand.cs" />
+    <Compile Include="Models.Generated\**\*.cs" />
```

Comment:
What is in Models.Generated??

Reply:
All of our `PS*` objects, i.e. `PSCloudTask`, etc. We have a basic tool which creates wrappers from them based on the C# SDK.